scripts/run-system-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
As this script depends on environment variables, I think it would be good to document it the same way as other scripts are documented in this directory.

Reply:
Right, added